security/access_token.rst

```diff
@@ -0,0 +1,347 @@
+`JSON Web Tokens (JWT)`_ or `SAML2 (XML structures)`_. Please refer to the
+`RFC6750`_: *The OAuth 2.0 Authorization Framework: Bearer Token Usage* for
+a detailed specification.
+
+Using the Access Token Authenticator
+------------------------------------
+
+This guide assumes you have setup security and have created a user object
+in your application. Follow :doc:`the main security guide </security>` if
+this is not yet the case.
+
+1) Configure the Access Token Authenticator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To use the access token authenticator, you must configure a ``token_handler``.
+The token handler receives the token from the request and returns the
+correct user identifier. To get the user identifier, implementations may
+need to load and validate the token (e.g. revocation, expiration time,
+digital signature, etc.).
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/security.yaml
+        security:
+            firewalls:
+                main:
+                    access_token:
+                        token_handler: App\Security\AccessTokenHandler
+
+    .. code-block:: xml
+
+        <!-- config/packages/security.xml -->
+        <?xml version="1.0" encoding="UTF-8"?>
+        <srv:container xmlns="http://symfony.com/schema/dic/security"
+            xmlns:srv="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/security
+                https://symfony.com/schema/dic/security/security-1.0.xsd">
+
+            <config>
+                <firewall name="main">
+                    <access-token token-handler="App\Security\AccessTokenHandler"/>
+                </firewall>
+            </config>
+        </srv:container>
+
+    .. code-block:: php
+
+        // config/packages/security.php
+        use App\Security\AccessTokenHandler;
+        use Symfony\Config\SecurityConfig;
+
+        return static function (SecurityConfig $security) {
+            $security->firewall('main')
+                ->accessToken()
+                    ->tokenHandler(AccessTokenHandler::class)
+            ;
+        };
+
+This handler must implement
+:class:`Symfony\\Component\\Security\\Http\\AccessToken\\AccessTokenHandlerInterface`::
+
+    // src/Security/AccessTokenHandler.php
+    namespace App\Security;
+
+    use App\Repository\AccessTokenRepository;
+    use Symfony\Component\Security\Http\AccessToken\AccessTokenHandlerInterface;
+
+    class AccessTokenHandler implements AccessTokenHandlerInterface
+    {
+        public function __construct(
+            private AccessTokenRepository $repository
+        ) {
+        }
+
+        public function getUserIdentifierFrom(string $token): string
```

Comment:
Since https://github.com/symfony/symfony/pull/48285, the AccessTokenHandlerInterface has been updated as following:
```suggestion
        public function getUserBadgeFrom(string $accessToken): UserBadge
```